### ISSUE TITLE
Taxonomies are not synced to new translation of media

### ISSUE THREAD

**Issue by chesio** [polylang/polylang#72]
Hi Chouby,

I have an install with custom taxonomies for attachment post type (ie media). I noticed that when I create new translation of media, taxonomies from existing translation are not synced. However, they are synced on update.

My settings are:
- media translation active, automatic duplication off
- my media taxonomy is public and have translations managed by Polylang
- post taxonomies are set to be synced

I'm using Polylang Pro 2.0.10 on WordPress 4.7.

**Comment by Chouby** [comment]
Hi,

This is caused by a conflict with the taxonomy synchronization and the shared post slugs. Changing this priority in modules/share-slug/share-post-slug.php should fix the issue:

```
@@ -20,7 +20,7 @@
 		$this->links_model = &$polylang->links_model;
 
 		add_filter( 'wp_unique_post_slug', array( $this, 'wp_unique_post_slug' ), 10, 6 );
-		add_action( 'pll_translate_media', array( $this, 'pll_translate_media' ), 10, 3 );
+		add_action( 'pll_translate_media', array( $this, 'pll_translate_media' ), 20, 3 ); // After PLL_Admin_Sync to avoid reverse sync
 	}
 
 	/**
```

**Comment by chesio** [comment]
I can confirm that your fix works, thanks! Is this also going to be included in 2.0.11?

**Comment by Chouby** [comment]
Perfect. Yes I will include this in 2.0.11 too.
